prowler/providers/aws/services/elb/elb_service.py

```diff
@@ -7,17 +7,16 @@
 from prowler.providers.aws.lib.service.service import AWSService
 
 
-################### ELB
 class ELB(AWSService):
     def __init__(self, provider):
         # Call AWSService's __init__
         super().__init__(__class__.__name__, provider)
-        self.loadbalancers = []
-        self.__threading_call__(self.__describe_load_balancers__)
-        self.__threading_call__(self.__describe_load_balancer_attributes__)
-        self.__describe_tags__()
+        self.loadbalancers = {}
+        self.__threading_call__(self._describe_load_balancers)
+        self.__threading_call__(self._describe_load_balancer_attributes)
```

Comment:
Please, could you use a thread for each load balancer using?
```suggestion
        self.__threading_call__(self._describe_load_balancer_attributes, self.loadbalancers)
```